OBSDOCS-462: Update logging uninstall docs (#69255)
openshift/openshift-docs: modules/uninstall-es-operator.adoc

// Module included in the following assemblies:
//
// * logging/cluster-logging-uninstall.adoc

:_mod-docs-content-type: PROCEDURE
[id="uninstall-es-operator_{context}"]
= Uninstalling Elasticsearch

.Prerequisites

* You have administrator permissions.
* You have access to the *Administrator* perspective of the {product-title} web console.
* If you have not already removed the {clo} and related resources, you must remove references to Elasticsearch from the `ClusterLogging` custom resource.

.Procedure

> . Navigate to the *Administration* -> *Custom Resource Definitions* page. Click *Elasticsearch*.

Reviewer: ```suggestion
. Go to the *Administration* -> *Custom Resource Definitions* page, and click *Elasticsearch*.
```

. On the *Custom Resource Definition Details* page, click *Instances*.

. Click the options menu {kebab} next to the instance, then click *Delete Elasticsearch*.

> . Navigate to the *Administration* -> *Custom Resource Definitions* page.

Reviewer: ```suggestion
. Go to the *Administration* -> *Custom Resource Definitions* page.
```

. Click the options menu {kebab} next to *Elasticsearch* and select *Delete Custom Resource Definition*.

. Delete the object storage secret.

. Navigate to the *Operators* -> *Installed Operators* page.

. Click the options menu {kebab} next to the {es-op}, then click *Uninstall Operator*.

. Optional: Delete the `openshift-operators-redhat` project.
+
[IMPORTANT]
====
Do not delete the `openshift-operators-redhat` project if other global Operators are installed in this namespace.
====

.. Switch to the *Home* -> *Projects* page.

> .. Click the options menu {kebab} next to the *openshift-operators-redhat* project, then click *Delete Project*.

Reviewer: ```suggestion
.. Click the options menu {kebab} next to the *openshift-operators-redhat* project, and then click *Delete Project*.
```
